src/psyclone/psyir/nodes/container.py

```diff
@@ -123,10 +122,7 @@ def create(cls, name, symbol_table, children):
                 "should be a list but found '{0}'."
                 "".format(type(children).__name__))
 
-        container = cls(name)
-        # pylint: disable=protected-access
-        container._symbol_table = symbol_table
-        symbol_table._node = container
+        container = cls(name, symbol_table=symbol_table)
```

Comment:
Please could you remove the uses of `format()` in this method.

Reply:
Done